src/platform/site-wide/sass/merger.scss

```diff
@@ -2,6 +2,10 @@
 
 /* OVERRIDE */
 .merger {
+  .va-dropdown-trigger {
+    min-width: 102px;
```

Comment:
not a fan of magic numbers but this was what I came up with without altering the markup. The drop down is in a separate container from the button so it's tricky to dynamically get this width without js.